pyteal/ast/abi/tuple_test.py

```diff
@@ -371,6 +372,15 @@ class IndexTest(NamedTuple):
                 encoded, start_index=pt.ExtractUint16(encoded, pt.Int(0))
             ),
         ),
+        IndexTest(
+            types=[dynamic_array_t1, static_bytes_t],
+            typeIndex=1,
+            expected=lambda output: output.decode(
+                encoded,
+                start_index=pt.Int(2),
+                length=pt.Int(2),
+            ),
+        ),
         IndexTest(
```

Comment:
Could you also add a test case to add coverage for indexing the last element of static tuples? I think something like this would work (but not tested):

```python
        IndexTest(
            types=[static_bytes_t, static_bytes_t],
            typeIndex=1,
            expected=lambda output: output.decode(
                encoded,
                start_index=pt.Int(2),
            ),
        ),
```

Reply:
yes, the testcase your proposed is correct, see [8fb2e33](https://github.com/algorand/pyteal/pull/663/commits/8fb2e33f8448ddd58668855a2a71df0982a7678b)